interface/super/edit_layout.php

```diff
@@ -580,7 +580,9 @@ class='optin noselect' style='width:100%' />";
           "A = " . xla('Age') .
         ", B = " . xla('Gestational Age') .
         ", C = " . xla('Capitalize') .
-        ", D = " . xla('Dup Check') .
+        ", D = " . xla('Dup Check New') .
+        ", E = " . xla('Dup Check addon Edit') .
+        ", W = " . xla('Dup Check addon New') .
```

Comment:
Wouldn't the following make more sense:
``````
        ", D = " . xla('Dup Check') .
        ", E = " . xla('Dup Check on only Edit') .
        ", W = " . xla('Dup Check on only New')
``````
Since it appears D means dup check is always done whereas the other are for specific cases.